### ISSUE TITLE
The loss is not 0.3, even after 100epochs

### ISSUE THREAD

**Issue by SashoNedelkoski** [booz-allen-hamilton/DSB3Tutorial#11]
Hi, I want to ask if maybe there are an errors in the code, because the loss never approach 0.3 as stated in the tutorial...

**Comment by jonrmulholland** [comment]
I can't guarantee that there aren't errors in the code as posted. This repo was put together in a hurry and combines the work streams of two people that were initially working independently. 

I'm afraid that I can't really help troubleshooting the issue here. Please think of the code here as an outline of an approach rather than as a benchmark. 

There shouldn't be too much room for error in the unet and training code though, so maybe double check that the training data and masks look right and that the pixel values are in the correct range.

**Comment by dszhengyu** [comment]
the pixel values among train set (lung roi as X, nodule mask as Y) should be between [0, 1), right?

**Comment by abhiML** [comment]
@dszhengyu Did you find a solution?

**Comment by aszhanghuali** [comment]
@dszhengyu  Hi!Did you run classify_nodes.py successfully ?
I run this successfully ,but It's all 0 in dataY.npy. Why is there no 1?
Looking forward to your reply!

**Comment by ZhongXing7695** [comment]
@aszhanghuali Hi!How do you run the classify_nodes.py successful? I feel some files missing. Would you send me the source code and files? I really want to run the program completely. My email is 877863872@qq.com. Thx very much.

**Comment by aszhanghuali** [comment]
这个代码不全，并且文件也不完善，没有成功就放弃了


------------------ 原始邮件 ------------------
发件人: "ZhongXing7695"<notifications@github.com>;
发送时间: 2019年6月14日(星期五) 上午9:36
收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>;
抄送: "红楼小梦"<1319466165@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11)




@aszhanghuali Hi!How do you run the classify_nodes.py successful? I feel some files missing. Would you send me the source code and files? I really want to run the program completely. My email is 877863872@qq.com. Thx very much.

**Comment by ZhongXing7695** [comment]
你也是前面三个文件跑通了，最后一个分类的没有跑通吗？



------------------ 原始邮件 ------------------
发件人: "as15538191859"<notifications@github.com>;
发送时间: 2019年6月14日(星期五) 上午9:38
收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>;
抄送: "仲兴"<877863872@qq.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11)



这个代码不全，并且文件也不完善，没有成功就放弃了 
 
 
 ------------------ 原始邮件 ------------------ 
 发件人: "ZhongXing7695"<notifications@github.com>; 
 发送时间: 2019年6月14日(星期五) 上午9:36 
 收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 
 抄送: "红楼小梦"<1319466165@qq.com>;"Mention"<mention@noreply.github.com>; 
 主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 
 
 
 
 
 @aszhanghuali Hi!How do you run the classify_nodes.py successful? I feel some files missing. Would you send me the source code and files? I really want to run the program completely. My email is 877863872@qq.com. Thx very much.

**Comment by aszhanghuali** [comment]
是的




------------------ 原始邮件 ------------------
发件人: "ZhongXing7695"<notifications@github.com>;
发送时间: 2019年6月14日(星期五) 上午9:40
收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>;
抄送: "红楼小梦"<1319466165@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11)



你也是前面三个文件跑通了，最后一个分类的没有跑通吗？ 
 
 
 
 ------------------ 原始邮件 ------------------ 
 发件人: "as15538191859"<notifications@github.com>; 
 发送时间: 2019年6月14日(星期五) 上午9:38 
 收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 
 抄送: "仲兴"<877863872@qq.com>;"Comment"<comment@noreply.github.com>; 
 主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 
 
 
 
 这个代码不全，并且文件也不完善，没有成功就放弃了  
   
   
  ------------------ 原始邮件 ------------------  
  发件人: "ZhongXing7695"<notifications@github.com>;  
  发送时间: 2019年6月14日(星期五) 上午9:36  
  收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>;  
  抄送: "红楼小梦"<1319466165@qq.com>;"Mention"<mention@noreply.github.com>;  
  主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11)  
   
   
   
   
  @aszhanghuali Hi!How do you run the classify_nodes.py successful? I feel some files missing. Would you send me the source code and files? I really want to run the program completely. My email is 877863872@qq.com. Thx very much.

**Comment by ZhongXing7695** [comment]
好的，谢谢。有可能的能加我一下好友吗？以后可以多多交流。
  

 

 ------------------ 原始邮件 ------------------
  发件人: "as15538191859"<notifications@github.com>;
 发送时间: 2019年6月14日(星期五) 上午9:41
 收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>;
 抄送: "仲兴"<877863872@qq.com>;"Comment"<comment@noreply.github.com>;
 主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11)

 

是的 




------------------ 原始邮件 ------------------ 
发件人: "ZhongXing7695"<notifications@github.com>; 
发送时间: 2019年6月14日(星期五) 上午9:40 
收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 
抄送: "红楼小梦"<1319466165@qq.com>;"Mention"<mention@noreply.github.com>; 
主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 



你也是前面三个文件跑通了，最后一个分类的没有跑通吗？ 



------------------ 原始邮件 ------------------ 
发件人: "as15538191859"<notifications@github.com>; 
发送时间: 2019年6月14日(星期五) 上午9:38 
收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 
抄送: "仲兴"<877863872@qq.com>;"Comment"<comment@noreply.github.com>; 
主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 



这个代码不全，并且文件也不完善，没有成功就放弃了 


------------------ 原始邮件 ------------------ 
发件人: "ZhongXing7695"<notifications@github.com>; 
发送时间: 2019年6月14日(星期五) 上午9:36 
收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 
抄送: "红楼小梦"<1319466165@qq.com>;"Mention"<mention@noreply.github.com>; 
主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 




@aszhanghuali Hi!How do you run the classify_nodes.py successful? I feel some files missing. Would you send me the source code and files? I really want to run the program completely. My email is 877863872@qq.com. Thx very much.

**Comment by aszhanghuali** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        好
        
    
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                1319466165
                            
                    
                        
                            
                                    1319466165@qq.com
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 6/14/2019 11:23，ZhongXing7695<notifications@github.com> wrote： 


好的，谢谢。有可能的能加我一下好友吗？以后可以多多交流。



  







 







 ------------------ 原始邮件 ------------------



  发件人: "as15538191859"<notifications@github.com>;



 发送时间: 2019年6月14日(星期五) 上午9:41



 收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>;



 抄送: "仲兴"<877863872@qq.com>;"Comment"<comment@noreply.github.com>;



 主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11)







 







是的 



















------------------ 原始邮件 ------------------ 



发件人: "ZhongXing7695"<notifications@github.com>; 



发送时间: 2019年6月14日(星期五) 上午9:40 



收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 



抄送: "红楼小梦"<1319466165@qq.com>;"Mention"<mention@noreply.github.com>; 



主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 















你也是前面三个文件跑通了，最后一个分类的没有跑通吗？ 















------------------ 原始邮件 ------------------ 



发件人: "as15538191859"<notifications@github.com>; 



发送时间: 2019年6月14日(星期五) 上午9:38 



收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 



抄送: "仲兴"<877863872@qq.com>;"Comment"<comment@noreply.github.com>; 



主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 















这个代码不全，并且文件也不完善，没有成功就放弃了 











------------------ 原始邮件 ------------------ 



发件人: "ZhongXing7695"<notifications@github.com>; 



发送时间: 2019年6月14日(星期五) 上午9:36 



收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 



抄送: "红楼小梦"<1319466165@qq.com>;"Mention"<mention@noreply.github.com>; 



主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 



















@aszhanghuali Hi!How do you run the classify_nodes.py successful? I feel some files missing. Would you send me the source code and files? I really want to run the program completely. My email is 877863872@qq.com. Thx very much. 







— 



You are receiving this because you were mentioned. 



Reply to this email directly, view it on GitHub, or mute the thread. 



— 



You are receiving this because you commented. 



Reply to this email directly, view it on GitHub, or mute the thread. 



— 



You are receiving this because you were mentioned. 



Reply to this email directly, view it on GitHub, or mute the thread.  



—



You are receiving this because you commented.



Reply to this email directly, view it on GitHub, or mute the thread.



—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by ZhongXing7695** [comment]
实在不好意思，你的QQ有验证消息，需要填写你的真实姓名？我加不了哎，要不你添加我的？

 

 ------------------ 原始邮件 ------------------
  发件人: "as15538191859"<notifications@github.com>;
 发送时间: 2019年6月14日(星期五) 中午11:24
 收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>;
 抄送: "仲兴"<877863872@qq.com>;"Comment"<comment@noreply.github.com>;
 主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11)

 







font{ 
line-height: 1.6; 
} 
ul,ol{ 
padding-left: 20px; 
list-style-position: inside; 
} 




好



















1319466165




1319466165@qq.com








签名由
网易邮箱大师
定制



On 6/14/2019 11:23，ZhongXing7695<notifications@github.com> wrote： 


好的，谢谢。有可能的能加我一下好友吗？以后可以多多交流。



















------------------ 原始邮件 ------------------



发件人: "as15538191859"<notifications@github.com>;



发送时间: 2019年6月14日(星期五) 上午9:41



收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>;



抄送: "仲兴"<877863872@qq.com>;"Comment"<comment@noreply.github.com>;



主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11)















是的 



















------------------ 原始邮件 ------------------ 



发件人: "ZhongXing7695"<notifications@github.com>; 



发送时间: 2019年6月14日(星期五) 上午9:40 



收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 



抄送: "红楼小梦"<1319466165@qq.com>;"Mention"<mention@noreply.github.com>; 



主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 















你也是前面三个文件跑通了，最后一个分类的没有跑通吗？ 















------------------ 原始邮件 ------------------ 



发件人: "as15538191859"<notifications@github.com>; 



发送时间: 2019年6月14日(星期五) 上午9:38 



收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 



抄送: "仲兴"<877863872@qq.com>;"Comment"<comment@noreply.github.com>; 



主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 















这个代码不全，并且文件也不完善，没有成功就放弃了 











------------------ 原始邮件 ------------------ 



发件人: "ZhongXing7695"<notifications@github.com>; 



发送时间: 2019年6月14日(星期五) 上午9:36 



收件人: "booz-allen-hamilton/DSB3Tutorial"<DSB3Tutorial@noreply.github.com>; 



抄送: "红楼小梦"<1319466165@qq.com>;"Mention"<mention@noreply.github.com>; 



主题: Re: [booz-allen-hamilton/DSB3Tutorial] The loss is not 0.3, evenafter 100epochs (#11) 



















@aszhanghuali Hi!How do you run the classify_nodes.py successful? I feel some files missing. Would you send me the source code and files? I really want to run the program completely. My email is 877863872@qq.com. Thx very much.
